library-best-practices/glossary.md

```diff
@@ -19,50 +19,26 @@ Java Dependency Glossary
 ### Types of conflicts and compatibility
 
 <a name="linkage-error"></a>
-<a name="static-linkage-error"></a>
-<a name="dynamic-linkage-error"></a>
-- **Linkage error**: an error when a Java class references
-  another class and the reference cannot be satisfied with the available classes in the class path.
+- **Linkage error**: A "class has some dependency on another class; however, the
```

Comment:
I felt the double quotation typo.
![image](https://user-images.githubusercontent.com/28604/52369784-6b0ede00-2a1f-11e9-9929-e78828e70616.png)

Also would you define this as a noun, not a quoted sentence? For example "an error when ..."

Reply:
How about 

> An unsatisfactory dependency where "class has some dependency on another ....

?

Reply:
fixed